packages/vscode-dependencies-validation/src/commands.ts

```diff
@@ -0,0 +1,40 @@
+import { dirname } from "path";
+import type { DiagnosticCollection } from "vscode";
+import { invokeNPMCommand } from "@sap-devx/npm-dependencies-validation";
+import { refreshDiagnostics } from "./diagnostics";
+import { VscodeOutputChannel } from "./vscodeTypes";
+
+export async function fixAllDepIssuesCommand(
+  outputChannel: VscodeOutputChannel,
+  packageJsonPath: string,
+  dependencyIssuesDiagnosticCollection: DiagnosticCollection
+): Promise<void> {
+  outputChannel.show(true);
+
+  const npmCommand = "install";
+  try {
+    outputChannel.appendLine(`\nFixing dependency issues ...`);
+    const start = Date.now();
+    const config = { commandArgs: [npmCommand], cwd: dirname(packageJsonPath) };
+    await invokeNPMCommand(config, outputChannel);
```

Comment:
invokeNPMCommmand prints everyting to output channel as a single JSON line.
but that is hardly human readable.
we should probably pass the outputChannel line by line instead, and not in JSON format.

Reply:
I think it is not possible because we run the "npm ls --depth=0" with --json option.